### DISCUSSION TITLE
Интеграция с базами данных ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#49] [Announcements]
# Интеграция с базами данных ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/expressjs4/database-integration/

**Comment by noxaoc** [comment]
Непонятен момент как при асинхронной sqlite3 отдать данные на get - запрос, к примеру.

**Comment by sorokaIlya** [comment]
вообще круто было бы добавить TypeOrm. Вообще если рассмотреть как node js подойдет с orm для веб приложений.
